ctrtool/romfs.c

```diff
@@ -359,7 +359,7 @@ void romfs_extract_datafile(romfs_context* ctx, u64 offset, u64 size, const osch
 	outfile = os_fopen(path, OS_MODE_WRITE);
 	if (outfile == NULL)
 	{
-		fprintf(stderr, "Error opening file for writing\n");
+		fprintf(stderr, "Error opening file %s for writing\n", path);
```

Comment:
We can't use fprintf with `oschar_t` strings because on windows oschar_t is `typedef wchar_t oschar_t; // UTF16-LE`.

This would work though:
```
fprintf(stderr, "Error opening file ");
os_fputs(path, stderr);
fprintf(stderr, "for writing\n");
```

Reply:
Sorry, that doesn't even belong in this patch altogether, it's just debug residue I didn't get rid of.